include/nbl/video/IPhysicalDevice.h

```diff
@@ -113,7 +113,13 @@ class IPhysicalDevice : public core::Interface, public core::Unmovable
     virtual IDebugCallback* getDebugCallback() = 0;
 
     virtual bool isSwapchainSupported() const = 0;
+        
+    virtual bool isRayTracingPipelineSupported() const = 0;
 
+    virtual bool isAccelerationStructuresSupported() const = 0;
+
+    virtual bool isRayQuerySupported() const = 0;
```

Comment:
I'm a bit concerned about redundancy here, lots and lots of code to find out what's already present in 
```cpp
    const SLimits& getLimits() const { return m_limits; }
    const SFeatures& getFeatures() const { return m_features; }
```
anyway

Reply:
Do you want me to remove these functions?  @devshgraphicsprogramming

Reply:
yes, would make most sense.